data/free worlds start.txt

```diff
@@ -1399,6 +1399,98 @@ mission "FW Pirates 3.1B"
 
 
 
+mission "FW Pirates 3.2C"
+	name "Crush the Pirates of Greenrock"
+	description "Destroy the combined pirate armada while they're concentrated in this system, then put an end to their way of life on Greenrock."
+	blocked <message>
+	deadline 0
+	deadline 1
```

Comment:
I think you want `deadline 1 0` instead of two lines.
(1 + 0*hyperspace jumps to destination) = 1 day deadline.